client/app/_users/users.html

```diff
@@ -67,7 +67,12 @@
                                                 <b name="userUsername">{{user.username}}</b>
                                             </td>
                                             <td data-title="Email" name="userEmail" md-cell>{{user.email}}</td>
-                                            <td data-title="First/Last name" name="userFirstLastName" md-cell>{{user.firstName}} {{user.lastName}}</td>
+                                            <td class="users-table-row _name" data-title="First/Last name" name="userFirstLastName" md-cell>
```

Comment:
please use `users-table__row` instead of `users-table-row`

Reply:
fixed